### PR TITLE
Scripts to create "candidates" and GTs for Scenario 2021

### PR DESCRIPTION
**Pull request by slomeo**
#### PR description:

Scripts to create "candidates" and GTs for Scenario 2021

#### PR validation:

test ok by using Scenario 2021 (see PR #27744 ): no overlaps inside Muon System and all MB4Shields for Run3
candidate name: 106X_mcRun3_2021_realistic_Candidate_2019_08_12_09_16_22
candidate label: Extended2021
from GT: 106X_mcRun3_2021_realistic_Queue
scenario: RunIII Simulation

### CODE REVIEW

**Comment by cvuosalo** [comment]
@cmsbuild please test

File: CondTools/Geometry/test/geometryCTPPStest_local.py [hunk 1]
```diff
@@ -13,7 +13,7 @@
 
 process.GlobalTag.toGet = cms.VPSet(
     cms.PSet(record = cms.string('GeometryFileRcd'),
-             tag = cms.string('XMLFILE_Geometry_TagXX_Extended2017_mc'),
+             tag = cms.string('XMLFILE_Geometry_110YV1_Extended2017_mc'),
```
Comment (ianna) [inline CondTools/Geometry/test/geometryCTPPStest_local.py]: TagXX is replaced by a script, please do not change it

File: CondTools/Geometry/test/geometrytest_local.py [hunk 2]
```diff
@@ -13,65 +13,65 @@
 
 process.GlobalTag.toGet = cms.VPSet(
     cms.PSet(record = cms.string("PCastorRcd"),
-             tag = cms.string("CASTORRECO_Geometry_TagXX"),
+             tag = cms.string("CASTORRECO_Geometry_110YV1"),
```
Comment (ianna) [inline CondTools/Geometry/test/geometrytest_local.py]: TagXX is replaced by a script, please do not change it

File: CondTools/Geometry/test/geometrytest_local.py [hunk 3]
```diff
@@ -13,65 +13,65 @@
...
              connect = cms.string("sqlite_file:myfile.db")
              ),
     cms.PSet(record = cms.string("PZdcRcd"),
-             tag = cms.string("ZDCRECO_Geometry_TagXX"),
+             tag = cms.string("ZDCRECO_Geometry_110YV1"),
```
Comment (ianna) [inline CondTools/Geometry/test/geometrytest_local.py]: TagXX is replaced by a script, please do not change it

File: CondTools/Geometry/test/geometrytest_local.py [hunk 4]
```diff
@@ -13,65 +13,65 @@
...
              connect = cms.string("sqlite_file:myfile.db")
              ),
     cms.PSet(record = cms.string("PCaloTowerRcd"),
-             tag = cms.string("CTRECO_Geometry_TagXX"),
+             tag = cms.string("CTRECO_Geometry_110YV1"),
```
Comment (ianna) [inline CondTools/Geometry/test/geometrytest_local.py]: TagXX is replaced by a script, please do not change it

File: CondTools/Geometry/test/geometrytest_local.py [hunk 5]
```diff
@@ -13,65 +13,65 @@
...
              connect = cms.string("sqlite_file:myfile.db")
              ),
     cms.PSet(record = cms.string("PEcalEndcapRcd"),
-             tag = cms.string("EERECO_Geometry_TagXX"),
+             tag = cms.string("EERECO_Geometry_110YV1"),
```
Comment (ianna) [inline CondTools/Geometry/test/geometrytest_local.py]: TagXX is replaced by a script, please do not change it

File: CondTools/Geometry/test/geometrytest_local.py [hunk 6]
```diff
@@ -13,65 +13,65 @@
...
              connect = cms.string("sqlite_file:myfile.db")
              ),
     cms.PSet(record = cms.string("CSCRecoDigiParametersRcd"),
-             tag = cms.string("CSCRECODIGI_Geometry_TagXX"),
+             tag = cms.string("CSCRECODIGI_Geometry_110YV1"),
```
Comment (ianna) [inline CondTools/Geometry/test/geometrytest_local.py]: TagXX is replaced by a script, please do not change it

File: CondTools/Geometry/test/geometrytest_local.py [hunk 7]
```diff
@@ -13,65 +13,65 @@
...
              connect = cms.string("sqlite_file:myfile.db")
              ),
     cms.PSet(record = cms.string("CSCRecoGeometryRcd"),
-             tag = cms.string("CSCRECO_Geometry_TagXX"),
+             tag = cms.string("CSCRECO_Geometry_110YV1"),
```
Comment (ianna) [inline CondTools/Geometry/test/geometrytest_local.py]: TagXX is replaced by a script, please do not change it

File: CondTools/Geometry/test/geometrytest_local.py [hunk 8]
```diff
@@ -13,65 +13,65 @@
...
              connect = cms.string("sqlite_file:myfile.db")
              ),
     cms.PSet(record = cms.string("PEcalBarrelRcd"),
-             tag = cms.string("EBRECO_Geometry_TagXX"),
+             tag = cms.string("EBRECO_Geometry_110YV1"),
```
Comment (ianna) [inline CondTools/Geometry/test/geometrytest_local.py]: TagXX is replaced by a script, please do not change it

File: CondTools/Geometry/test/geometrytest_local.py [hunk 9]
```diff
@@ -13,65 +13,65 @@
...
              connect = cms.string("sqlite_file:myfile.db")
              ),
     cms.PSet(record = cms.string("RPCRecoGeometryRcd"),
-             tag = cms.string("RPCRECO_Geometry_TagXX"),
+             tag = cms.string("RPCRECO_Geometry_110YV1"),
```
Comment (ianna) [inline CondTools/Geometry/test/geometrytest_local.py]: TagXX is replaced by a script, please do not change it

File: CondTools/Geometry/test/geometrytest_local.py [hunk 10]
```diff
@@ -13,65 +13,65 @@
...
              connect = cms.string("sqlite_file:myfile.db")
              ),
     cms.PSet(record = cms.string("DTRecoGeometryRcd"),
-             tag = cms.string("DTRECO_Geometry_TagXX"),
+             tag = cms.string("DTRECO_Geometry_110YV1"),
```
Comment (ianna) [inline CondTools/Geometry/test/geometrytest_local.py]: TagXX is replaced by a script, please do not change it

File: CondTools/Geometry/test/geometrytest_local.py [hunk 11]
```diff
@@ -13,65 +13,65 @@
...
              connect = cms.string("sqlite_file:myfile.db")
              ),
     cms.PSet(record = cms.string("PEcalPreshowerRcd"),
-             tag = cms.string("EPRECO_Geometry_TagXX"),
+             tag = cms.string("EPRECO_Geometry_110YV1"),
```
Comment (ianna) [inline CondTools/Geometry/test/geometrytest_local.py]: TagXX is replaced by a script, please do not change it

File: CondTools/Geometry/test/geometrytest_local.py [hunk 12]
```diff
@@ -13,65 +13,65 @@
...
              connect = cms.string("sqlite_file:myfile.db")
              ),
     cms.PSet(record = cms.string("GeometryFileRcd"),
-             tag = cms.string("XMLFILE_Geometry_Extended_TagXX"),
+             tag = cms.string("XMLFILE_Geometry_Extended_110YV1"),
```
Comment (ianna) [inline CondTools/Geometry/test/geometrytest_local.py]: TagXX is replaced by a script, please do not change it

File: CondTools/Geometry/test/geometrytest_local.py [hunk 13]
```diff
@@ -13,65 +13,65 @@
...
              connect = cms.string("sqlite_file:myfile.db"),
 #             label = cms.string("Extended")
              ),
     cms.PSet(record = cms.string("PGeometricDetExtraRcd"),
-             tag = cms.string("TKExtra_Geometry_TagXX"),
+             tag = cms.string("TKExtra_Geometry_110YV1"),
```
Comment (ianna) [inline CondTools/Geometry/test/geometrytest_local.py]: TagXX is replaced by a script, please do not change it

File: CondTools/Geometry/test/geometrytest_local.py [hunk 14]
```diff
@@ -13,65 +13,65 @@
...
              connect = cms.string("sqlite_file:myfile.db"),
 #             label = cms.string("Extra")
              ),
     cms.PSet(record = cms.string("IdealGeometryRecord"),
-             tag = cms.string("TKRECO_Geometry_TagXX"),
+             tag = cms.string("TKRECO_Geometry_110YV1"),
```
Comment (ianna) [inline CondTools/Geometry/test/geometrytest_local.py]: TagXX is replaced by a script, please do not change it

File: CondTools/Geometry/test/geometrytest_local.py [hunk 15]
```diff
@@ -13,65 +13,65 @@
...
              connect = cms.string("sqlite_file:myfile.db")
              ),
     cms.PSet(record = cms.string('PTrackerParametersRcd'),
-             tag = cms.string('TKParameters_Geometry_TagXX'),
+             tag = cms.string('TKParameters_Geometry_110YV1'),
```
Comment (ianna) [inline CondTools/Geometry/test/geometrytest_local.py]: TagXX is replaced by a script, please do not change it

File: CondTools/Geometry/test/geometrytest_local.py [hunk 16]
```diff
@@ -13,65 +13,65 @@
...
              connect = cms.string("sqlite_file:myfile.db")
              ),
     cms.PSet(record = cms.string("PHcalRcd"),
-             tag = cms.string("HCALRECO_Geometry_TagXX"),
+             tag = cms.string("HCALRECO_Geometry_110YV1"),
```
Comment (ianna) [inline CondTools/Geometry/test/geometrytest_local.py]: TagXX is replaced by a script, please do not change it

File: CondTools/Geometry/test/geometrywriter.py [hunk 17]
```diff
@@ -1,20 +1,30 @@
...
-#this and the outputservice below are set up to do the first insert of GeometryFile
+
+# This reads the big XML file and the only way to fill the
+# nonreco part of the database is to read this file.  It
+# somewhat duplicates the information read from the little
+# XML files, but there is no way to directly build the
+# DDCompactView from this.
 process.XMLGeometryWriter = cms.EDAnalyzer("XMLGeometryBuilder",
-                                           XMLFileName = cms.untracked.string("./geTagXX.xml"),
+                                           XMLFileName = cms.untracked.string("./geSingleBigFile.xml"),
```
Comment (ianna) [inline CondTools/Geometry/test/geometrywriter.py]: TagXX is replaced by a script, please do not change it

File: CondTools/Geometry/test/geometrywriter.py [hunk 18]
```diff
@@ -37,27 +48,27 @@
...
-                                                            cms.PSet(record = cms.string('PZdcRcd'),          tag = cms.string('ZDCRECO_Geometry_TagXX')),
-                                                            cms.PSet(record = cms.string('PCastorRcd'),       tag = cms.string('CASTORRECO_Geometry_TagXX')),
-                                                            cms.PSet(record = cms.string('CSCRecoGeometryRcd'),tag = cms.string('CSCRECO_Geometry_TagXX')),
-                                                            cms.PSet(record = cms.string('CSCRecoDigiParametersRcd'),tag = cms.string('CSCRECODIGI_Geometry_TagXX')),
-                                                            cms.PSet(record = cms.string('DTRecoGeometryRcd'),tag = cms.string('DTRECO_Geometry_TagXX')),
-                                                            cms.PSet(record = cms.string('RPCRecoGeometryRcd'),tag = cms.string('RPCRECO_Geometry_TagXX'))
+                                          toPut = cms.VPSet(cms.PSet(record = cms.string('GeometryFileRcd'),tag = cms.string('XMLFILE_Geometry_110YV2_Extended_mc')),
```
Comment (ianna) [inline CondTools/Geometry/test/geometrywriter.py]: TagXX is replaced by a script, please do not change it

File: CondTools/Geometry/test/geometryxmlwriter.py [hunk 19]
```diff
@@ -12,7 +13,7 @@
 
 process.BigXMLWriter = cms.EDAnalyzer("OutputDDToDDL",
                               rotNumSeed = cms.int32(0),
-                              fileName = cms.untracked.string("geTagXX.xml")
+                              fileName = cms.untracked.string("./geSingleBigFile.xml")
```
Comment (ianna) [inline CondTools/Geometry/test/geometryxmlwriter.py]: TagXX is replaced by a script, please do not change it

File: CondTools/Geometry/test/xmlgeometrywriter.py [hunk 20]
```diff
@@ -11,15 +12,15 @@
                             )
 
 process.XMLGeometryWriter = cms.EDAnalyzer("XMLGeometryBuilder",
-                                           XMLFileName = cms.untracked.string("./geTagXX.xml"),
+                                           XMLFileName = cms.untracked.string("./geSingleBigFile.xml"),
```
Comment (ianna) [inline CondTools/Geometry/test/xmlgeometrywriter.py]: TagXX is replaced by a script, please do not change it

File: CondTools/Geometry/test/xmlgeometrywriter.py [hunk 21]
```diff
@@ -11,15 +12,15 @@
...
                                            ZIP = cms.untracked.bool(True)
                                            )
 
 process.CondDB.timetype = cms.untracked.string('runnumber')
 process.CondDB.connect = cms.string('sqlite_file:myfile.db')
 process.PoolDBOutputService = cms.Service("PoolDBOutputService",
                                           process.CondDB,
-                                          toPut = cms.VPSet(cms.PSet(record = cms.string('GeometryFileRcd'),tag = cms.string('XMLFILE_Geometry_Extended_TagXX')))
+                                          toPut = cms.VPSet(cms.PSet(record = cms.string('GeometryFileRcd'),tag = cms.string('XMLFILE_Geometry_110YV2_Extended_mc')))
```
Comment (ianna) [inline CondTools/Geometry/test/xmlgeometrywriter.py]: TagXX is replaced by a script, please do not change it

**Review by ianna — CHANGES_REQUESTED** [review]

**Comment by ianna** [comment]
@slomeo - it looks like a duplicate of https://github.com/cms-sw/cmssw/pull/27693

@fabiocos, @ggovi - FYI

**Comment by slomeo** [comment]
@ianna Yes, it seems the same PR. Probably it's better to close #27749

**Pull request closed without merging**